packages/fields/src/types/CloudinaryImage/ImageBlock.js

```diff
@@ -92,7 +92,10 @@ export class ImageBlock extends Block {
 
   getMutationOperationResults({ context }) {
     return {
-      [this.path]: context._blockMeta[this.joinList][this.path],
+      [this.path]:
+        context._blockMeta &&
+        context._blockMeta[this.joinList] &&
+        context._blockMeta[this.joinList][this.path],
```

Comment:
In an upcoming PR, it's possible that this data is not hydrated (eg; creating a content value which has no cloudinary images), so this check is more thorough.